packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -0,0 +1,299 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconUnverified from 'terra-icon/lib/icon/IconDiamond';
+import Observation from './common/observation/_Observation';
+import observationPropShape from './proptypes/observationPropTypes';
+import ResultError from './common/other/_ResultError';
+import NoData from './common/other/_KnownNoData';
+import { isEmpty, checkIsStatusInError, ConditionalWrapper } from './common/utils';
+import styles from './ClinicalResult.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Result Object with the clinical result data. Example object structure listed above.
+   */
+  resultData: PropTypes.shape({
+    /**
+     *  Blood Pressure grouped result id
+     */
+    id: PropTypes.string,
+    /**
+     *  Systolic Result for blood pressure.
+     */
+    systolic: observationPropShape,
+    /**
+     *  Diastolic Result for blood pressure.
+     */
+    diastolic: observationPropShape,
+  }),
+  /**
+   * Visually hides the unit of measure when presented in a series of side-by-side columns of the same unit.
+   */
+  hideUnit: PropTypes.bool,
+  /**
+   * Whether or not the text should be truncated in display. Restricts clinical result details each to one line.
+   */
+  isTruncated: PropTypes.bool,
+  /**
+   * Override that shows an Error display. Used when there is a known error or problem when retrieving or assembling the clinical result data.
+   */
+  hasResultError: PropTypes.bool,
+  /**
+   * Override that shows a known "No Data" display. Used when there is known to be no value for a given clinical result concept at a specific datetime.
+   */
+  hasResultNoData: PropTypes.bool,
+  /**
+   * @private
+   * Used by Flowsheet Result Cell to hide icons because it displays them in different positions.
+   */
+  hideAccessoryDisplays: PropTypes.bool,
+  /**
+   * @private
+   * The intl object to be injected for translations.
+   */
+  intl: intlShape.isRequired,
+};
+
+const defaultProps = {
+  resultData: {},
```

Comment:
Let's remove this default. I think we can make  resultData a required prop

Reply:
See the new structure @emilyrohrbough